content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -341,6 +341,12 @@ Once the last finalizer is removed, the resource is actually removed from etcd.
 
 ### Make a dry run request
 
+{{< note >}}
+Dry run users need write authorization to use the modifying verb on the resource.
```

Comment:
Would it be simpler to say that authorization for dry run and non-dry run requests is identical, so in order to make a dry run version of a request, a user must be authorized to perform the non-dry run version?

Reply:
Oh yes that is clearer than "write permission." Updated!